### PR TITLE
added blog post dwh part 1

### PR DESCRIPTION
**Pull request by noacktino**

### CODE REVIEW

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: "A journey towards a custom data warehouse solution Part 1 : Getting Started"
```
Comment (rbordo) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: Please change it to:
A Journey Towards a Custom Data Warehouse Solution - Part 1: Getting Started

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+   metadata is semi-structured but also contains unstructured data such as a title, a description, and the content itself e.g. in text form.
+2. _User preferences_ are reading interests explicitly selected by the user and are used to personalize the content we deliver to the user. User preferences data is a form of
+   structured data.
+3. _User behavior_ includes information about the content consumed by the user and is used to improve the users’ experience with our product for example by means of collaborative
+   filtering. User behavior data is also a form of structured data.
+
+upday itself is a startup and a dynamic working environment with an agile development process. We use cloud services in order to avoid the commitment and costs of purchasing
+hardware products. This enables very fast development, allows us to perform cost efficient short-term experiments, and to pivot directions quickly.
+
+## DWH – a tentative definition
```
Comment (rbordo) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: Please use capital always in headings (DWH - A Tentative Definition)

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+approximately 5 - 15 years ago, the domain of data warehousing was marked by products from big companies like Oracle, IBM, or Microsoft. Usually a product provided by one of these
+vendors was installed on an expensive server farm. This infrastructure was then used to create a multidimensional data cube which was continuously and automatically processed
+overnight. This was generally very expensive, highly inflexible, and lead to a strong product-customer dependency. It is easy to assume that it is complicated to change such a
+production-ready DWH solution after it is been established. We were wary of getting caught in a situation like that. But as we know, information technology is a vibrant area and
+rapidly changing. In recent years, the possibilities for data warehouses and even data management platforms have changed dramatically. Since we were not familiar with
+state-of-the-art techniques and best practices of building a DWH, the amount of possible products and solutions was overwhelming. Therefore, we decided to engage a consultant
+with domain expertise to guide us the first steps.
+
+## Basic components
```
Comment (rbordo) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: capital...

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+<center>
+<picture>
+  	<a href="/images/blog/dwh_journey/dwh_components.png"><img src="/images/blog/dwh_journey/dwh_components.png" alt="DWH components" width="850"></a>
+   	<figcaption>Figure 1: DWH components</figcaption>
+</picture>
+</center>
+
+
+## Processing steps and tools
```
Comment (rbordo) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: capital

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+  into the staging area. Available ETL tools are: Talend, Etleap, Alteryx, Pentaho, and AWS Lambda.
+* Scheduling is used to trigger specific actions time-controlled. For example, it is useful to execute long running and complex calculations overnight and not during the main
+  business hours. Another example is consecutive actions where one action waits for the successful termination of another. Some scheduling tools are: Jenkins, Talend Enterprise,
+  and SOS-Scheduler.
+* Monitoring is required to be informed about the condition of the DWH and the states of individual actions. It is crucial to be alerted automatically in case of problems.
+  Possible monitoring tools we identified are: Splunk or DataDog.
+* Storage is mandatory for a data processing where the data is persistently accommodated. Some available storage tools are: Redshift, Snowflake, PostgreSQL, and Cassandra.
+* Finally, reporting is the purpose of a DWH where, based on the data held in the DWH, business cases are summarized and visualized.
+
+## Data Lake – why would we need one?
```
Comment (rbordo) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: capital

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+<center>
+<picture>
+    <a href="/images/blog/dwh_journey/query_service.png"><img src="/images/blog/dwh_journey/query_service.png" alt="Components of a data lake" width="850"></a>
+    <figcaption>Figure 2: Components of a data lake</figcaption>
+</picture>
+</center>
+
+
+## Adaption of processing steps
```
Comment (rbordo) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: capital

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+The data processing steps are similar to those described in the process chain above and are still valid regarding the data lake approach. In our use case, we adapt the
+process chain in two main ways: storage and interactive query service.
+
+* Storage: For this use case, S3 and HDFS are appropriate storage solutions.
+* Interactive Query Service: Query tools can be Hadoop, Spark, or Amazon Athena.
+
+## Summary and outlook
```
Comment (rbordo) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: capital

File: _posts/blog/2016-10-05-dwh-01.md [hunk 8]
```diff
@@ -23,3 +23,5 @@ Given all these different use cases, it was clearly __time for a big data soluti
 In a series of articles we would like to __share our experiences__ with you and invite you on our __journey towards a functional custom data warehouse solution__. See us fail, despair, hope, fail again and maybe finally succeed in our undertaking.
 
 ___Let our journey begin…___
+
+<a href="../dwh-part1_getting_started/">A journey towards a custom data warehouse solution Part 1 : Getting Started</a>
```
Comment (rbordo) [inline _posts/blog/2016-10-05-dwh-01.md]: Please change it to:
A Journey Towards a Custom Data Warehouse Solution - Part 1: Getting Started

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+description:
+modified:
+categories: blog
+author: tino_noack
+excerpt: This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data warehouse (DWH), describe components regarding our use case, and identify tools which can be applied for implementation. Finally, we introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
+---
+# Abstract
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: you don't need the word abstract. It's not an science article.

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 1 above)
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: what do you say if you put the "Part 1:..." on a separate line? 
`...solution <br/>Part 1: ...`

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: "A journey towards a custom data warehouse solution Part 1 : Getting Started"
+description:
+modified:
+categories: blog
+author: tino_noack
+excerpt: This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data warehouse (DWH), describe components regarding our use case, and identify tools which can be applied for implementation. Finally, we introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: this looks like a bit too much text and very formal. 
Since this is the "preview" text, make it more to the point. 
Something in the lines of:
"Our definition of a data warehouse is based on the data we have and on how we work with it. We want to share what our DWH is, what its components are and the tools applied for its implementation. Finally, we introduce a data lake as a more 'Big-Data-friendly' but provisional solution"

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+warehouse (DWH), describe components regarding our use case, and identify tools which can be applied for implementation. Finally, we introduce a data lake as a more
+'Big-Data-friendly' but provisional solution.
+
+# Getting Started
+
+upday is a news app which aggregates content from thousands of sources including trusted brands and bloggers into a unique and simple format. It is available in a variety of
+European countries in a wide range of languages. upday is designed to be an intelligent news aggregator which learns about the reading habits and preferences of each unique user.
+For that reason, we have to cope with a huge volume of incoming data from a vast number of sources. For example, we handle the following kinds of data:
+
+1. _Content metadata_ such as information about the content type, the publisher, and the publishing date is amongst others used for maintaining the publisher relationship. Content
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: put these with bold, not just italic, to emphasize them more.

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+   structured data.
+3. _User behavior_ includes information about the content consumed by the user and is used to improve the users’ experience with our product for example by means of collaborative
+   filtering. User behavior data is also a form of structured data.
+
+upday itself is a startup and a dynamic working environment with an agile development process. We use cloud services in order to avoid the commitment and costs of purchasing
+hardware products. This enables very fast development, allows us to perform cost efficient short-term experiments, and to pivot directions quickly.
+
+## DWH – a tentative definition
+
+Every beginning is hard, and we certainly did not know where to start. Thus, we asked ourselves; What is a _data warehouse (DWH)_? We decided to start with the first use case we
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: can you put all the quotes on separate lines? 
Check out how Tomek did it [here](https://upday.github.io/blog/reactive_frustrations_1/)

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+could already see ahead of us. As stated by Inmon: “_A data warehouse is a subject-oriented, integrated, nonvolatile and time-variant collection of data in support of management's
+decisions._” [^1]. A more natural definition is provided by Kimball: “_A data warehouse is a copy of transaction data specifically structured for querying and reporting._” [^2].
+However, these definitions are still very abstract and did not help us a lot to understand what exactly to start with. In our minds, and from a historical point of view
+approximately 5 - 15 years ago, the domain of data warehousing was marked by products from big companies like Oracle, IBM, or Microsoft. Usually a product provided by one of these
+vendors was installed on an expensive server farm. This infrastructure was then used to create a multidimensional data cube which was continuously and automatically processed
+overnight. This was generally very expensive, highly inflexible, and lead to a strong product-customer dependency. It is easy to assume that it is complicated to change such a
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: this is a quite big block of text. can you create 2 paragraphs out of it?

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+rapidly changing. In recent years, the possibilities for data warehouses and even data management platforms have changed dramatically. Since we were not familiar with
+state-of-the-art techniques and best practices of building a DWH, the amount of possible products and solutions was overwhelming. Therefore, we decided to engage a consultant
+with domain expertise to guide us the first steps.
+
+## Basic components
+
+In collaboration with the consultant, we identified basic components of a DWH for our use case: _sources_, _staging area_, _DWH core_, and _data mart_. The points and diagram below
+describe these components:
+
+1. Sources: We already introduced three kinds of data in the first section but we did not mention the data sources. The content metadata, in agreement with the content publisher,
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: Can you make "Sources", "Staging area" and "DWH core" also bold?

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+3. DWH core: It is planned to load the cleaned and transformed data into the DWH core. The DWH core is intended to contain the entire data for general purpose analysis. This
+   contrasts the previous definitions which describe a DWH as subject-oriented. We wanted to keep that door open because analysis purposes were not clearly defined at that stage
+   and keeping the platform flexible was important to support comprehensive analysis by data scientists and data mining experts.
+4. Data mart: For our use case, the data mart is the business layer which is subject-oriented and optimized for analysis purposes. The data is loaded from the DWH core via a
+   separated ETL process and contains aggregated information which can be queried by business analysts.
+
+<center>
+<picture>
+  	<a href="/images/blog/dwh_journey/dwh_components.png"><img src="/images/blog/dwh_journey/dwh_components.png" alt="DWH components" width="850"></a>
+   	<figcaption>Figure 1: DWH components</figcaption>
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: you don't need to put "Figure 1:".
Just leave it as "DWH components"

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 16]
```diff
@@ -0,0 +1,134 @@
...
+</center>
+
+
+## Processing steps and tools
+
+From the consultancy, we learned that a DWH can be implemented as a flexible process chain which is applied to collect data from different sources. The collected data is then
+stored in a central location and used to generate reports, create insights, and support business decision-making. We identified the following process steps: _extract, transform
+and load (ETL)_, _scheduling_, _monitoring_, _storage_, and _reporting_. As briefly described below, each of these processing steps can be covered by a variety of tools:
+
+* The ETL process is one of the most important processing steps of the DWH. It is applied to extract, transform, and load data from various sources
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: can you also bold words like "ETL process", "Scheduling" , "Monitoring", "Sorage" and "reporting" ?

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 17]
```diff
@@ -0,0 +1,134 @@
...
+  business hours. Another example is consecutive actions where one action waits for the successful termination of another. Some scheduling tools are: Jenkins, Talend Enterprise,
+  and SOS-Scheduler.
+* Monitoring is required to be informed about the condition of the DWH and the states of individual actions. It is crucial to be alerted automatically in case of problems.
+  Possible monitoring tools we identified are: Splunk or DataDog.
+* Storage is mandatory for a data processing where the data is persistently accommodated. Some available storage tools are: Redshift, Snowflake, PostgreSQL, and Cassandra.
+* Finally, reporting is the purpose of a DWH where, based on the data held in the DWH, business cases are summarized and visualized.
+
+## Data Lake – why would we need one?
+
+We have already gained more than 6 million monthly unique users and it seems that this number will keep growing. Thus, our data management process is clearly within the realm
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: " 6 million monthly unique users"  - is this a number that we are allowed to make public?

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 18]
```diff
@@ -0,0 +1,134 @@
...
+of Big Data. However, relational database systems are usually not well-suited to handle semi-structured and unstructured data as we have to deal with. Consequently, we also
+discussed a more 'Big-Data-friendly' solution which is known as a data lake [^3]. Stein and Morriso describe a data lake as follows: "_A repository for large quantities and
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: similar, put quotes on separate lines.

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 19]
```diff
@@ -0,0 +1,134 @@
...
+</picture>
+</center>
+
+
+## Adaption of processing steps
+
+The data processing steps are similar to those described in the process chain above and are still valid regarding the data lake approach. In our use case, we adapt the
+process chain in two main ways: storage and interactive query service.
+
+* Storage: For this use case, S3 and HDFS are appropriate storage solutions.
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: bold storage and Interactive Query Service

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 20]
```diff
@@ -0,0 +1,134 @@
...
+1. Sources: As described earlier, we have to deal with structured, semi-structured, and unstructured data.
+2. Data Repository: This is used to store the data heterogeneously. It preserves the original data fidelity extracted from the sources and the lineage of data transformation.
+3. Interactive Query Service: This is a service which accesses the data repository and transforms the data into a format that can be consumed by data science, data mining, and
+   business intelligence experts. The interactive query service can be a transformation rule which defines how to transform the data. This transformation rule might be used to
+   store the results persistently into the data repository or into another data base. It can be also used for exploring the raw data.
+
+<center>
+<picture>
+    <a href="/images/blog/dwh_journey/query_service.png"><img src="/images/blog/dwh_journey/query_service.png" alt="Components of a data lake" width="850"></a>
+    <figcaption>Figure 2: Components of a data lake</figcaption>
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: No need for "Figure 2" here either.

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 21]
```diff
@@ -0,0 +1,134 @@
...
+
+* Storage: For this use case, S3 and HDFS are appropriate storage solutions.
+* Interactive Query Service: Query tools can be Hadoop, Spark, or Amazon Athena.
+
+## Summary and outlook
+
+In this article we briefly introduced upday and described the data we deal with. Further on, we came up with a tentative definition of a DWH and gave a general introduction to
+basic components and processing steps regarding our use case. Finally, we discussed a data lake as a more 'Big-Data-friendly' solution.
+
+This article gives a general overview which is deepened in the next articles. We will describe the first iteration of our DWH implementation and express the selection of tools.
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: How about:
"Stay tuned for the next blog posts, where we will describe ... "  instead of the "This article gives a general overview which is deepened in the next articles. We will describe "

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 22]
```diff
@@ -0,0 +1,134 @@
...
+
+The data processing steps are similar to those described in the process chain above and are still valid regarding the data lake approach. In our use case, we adapt the
+process chain in two main ways: storage and interactive query service.
+
+* Storage: For this use case, S3 and HDFS are appropriate storage solutions.
+* Interactive Query Service: Query tools can be Hadoop, Spark, or Amazon Athena.
+
+## Summary and outlook
+
+In this article we briefly introduced upday and described the data we deal with. Further on, we came up with a tentative definition of a DWH and gave a general introduction to
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: Here you just copy-pasted the intro. How about gathering the info that matters, that the reader should care about. What do you think about something in the lines of:
"being an intelligent news aggregator shaped the components of our DWH: sources, staging area, DWH core, and data mart. It allowed us to create a process based on exact steps: extract, transform and load (ETL), scheduling, monitoring, storage, and reporting and drove us into the direction of a “Data Lake” - a more ‘Big-Data-friendly’ solution. " 
I just tried to gather the key words of your article here. Maybe you can "arrange" them better.

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 23]
```diff
@@ -0,0 +1,134 @@
...
+author: tino_noack
+excerpt: This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data warehouse (DWH), describe components regarding our use case, and identify tools which can be applied for implementation. Finally, we introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
+---
+# Abstract
+
+This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a
+brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: there's one thing that's missing here: why? why should the reader care? 
If I get it right (please correct me, if I'm wrong), we had a problem: "having to cope with a huge volume of data from a vast number of sources". 
The DWH is upday's solution to this. It's our way of solving this "coping" problem. 

So instead of "Getting Started" - you're rather saying what our problem was.

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 24]
```diff
@@ -0,0 +1,134 @@
...
+basic components and processing steps regarding our use case. Finally, we discussed a data lake as a more 'Big-Data-friendly' solution.
+
+This article gives a general overview which is deepened in the next articles. We will describe the first iteration of our DWH implementation and express the selection of tools.
+Moreover, we will explore the evolutionary development of the first DWH implementation and the experiences we gained during this period including trials and mistakes we have made.
+
+## References
+
+[^1]: W. H. Inmon. Building the Data Warehouse. John Wiley & Sons, Inc., 2002.
+[^2]: R. Kimball, Mary Ross. The Data Warehouse Toolkit. The Complete Guide to Dimensional Modeling. Wiley, 2013
+[^3]: D. Woods, http://www.forbes.com/sites/ciocentral/2011/07/21/big-data-requires-a-big-new-architecture/#70af770d1d75, Forbes 2011
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: put the name of the article under a link.

**Review by florina-muntenescu — CHANGES_REQUESTED** [review]

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 25]
```diff
@@ -0,0 +1,134 @@
...
+European countries in a wide range of languages. upday is designed to be an intelligent news aggregator which learns about the reading habits and preferences of each unique user.
+For that reason, we have to cope with a huge volume of incoming data from a vast number of sources. For example, we handle the following kinds of data:
+
+1. _Content metadata_ such as information about the content type, the publisher, and the publishing date is amongst others used for maintaining the publisher relationship. Content
+   metadata is semi-structured but also contains unstructured data such as a title, a description, and the content itself e.g. in text form.
+2. _User preferences_ are reading interests explicitly selected by the user and are used to personalize the content we deliver to the user. User preferences data is a form of
+   structured data.
+3. _User behavior_ includes information about the content consumed by the user and is used to improve the users’ experience with our product for example by means of collaborative
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: can you mention that the data is annonymous, that we can't identify the person based on that, and that it's not sold. 
Maybe rephrase this in a nice way to make all of these understood.

**Review by florina-muntenescu — CHANGES_REQUESTED** [review]

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 17 above)
Comment (florina-muntenescu) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: just checked with Andi - use "a few million", not an exact number

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 1 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 2 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 3 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 4 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 5 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 6 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 7 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-10-05-dwh-01.md (hunk 8 above)
Comment (noacktino) [reply _posts/blog/2016-10-05-dwh-01.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 1 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 9 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: deleted

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 11 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 12 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 13 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 14 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 15 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 16 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 17 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 18 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 20 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 19 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 24 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 25 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 23 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 21 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 22 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 10 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 26]
```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "A Journey Towards a Custom Data Warehouse Solution <br/> Part 1 : Getting Started"
+description:
+modified:
+categories: blog
+author: tino_noack
+excerpt: We would like to share our first approach towards a custom data warehouse solution. This is based on the kinds and the amount of data we have to process. We describe DWH components and identify tools which can be applied for implementation. We also introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: "This is based on the amount and on the type of data that needs processing."

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 27]
```diff
@@ -0,0 +1,153 @@
...
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
+---
+
+This is part one of a <a href="../dwh-01">series of articles</a> in which we would like to share our journey with you and our experiences towards a custom data warehouse solution.
+Part one starts with a brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: from what I see online, the industry tends to use "type of data" and not "kinds of data". Could you replace it?

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 28]
```diff
@@ -0,0 +1,153 @@
...
+modified:
+categories: blog
+author: tino_noack
+excerpt: We would like to share our first approach towards a custom data warehouse solution. This is based on the kinds and the amount of data we have to process. We describe DWH components and identify tools which can be applied for implementation. We also introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
+---
+
+This is part one of a <a href="../dwh-01">series of articles</a> in which we would like to share our journey with you and our experiences towards a custom data warehouse solution.
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: "share our journey and experiences towards building a custom data ... "

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md [hunk 29]
```diff
@@ -0,0 +1,153 @@
...
+layout: post
+title: "A Journey Towards a Custom Data Warehouse Solution <br/> Part 1 : Getting Started"
+description:
+modified:
+categories: blog
+author: tino_noack
+excerpt: We would like to share our first approach towards a custom data warehouse solution. This is based on the kinds and the amount of data we have to process. We describe DWH components and identify tools which can be applied for implementation. We also introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
```
Comment (florina-muntenescu) [inline _posts/blog/2016-12-22-dwh-part1_getting_started.md]: Change the date of the blog post to today, both in the name of the post, and here.

**Review by florina-muntenescu — APPROVED** [review]

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 26 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 29 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done

File: _posts/blog/2016-12-22-dwh-part1_getting_started.md (hunk 28 above)
Comment (noacktino) [reply _posts/blog/2016-12-22-dwh-part1_getting_started.md]: done
